examples/examples/exampleA/main.tf

```diff
@@ -0,0 +1,17 @@
+module full-example{
+    source = "github.com/terraform-docs/terraform-docs//examples"
```

Comment:
The double `//`? Probably fine, but something different always leads to the question "why".